### PR TITLE
Lean readme

### PR DESCRIPTION
**Pull request by ahmadiq**

### CODE REVIEW

**Comment by ahmadiq** [comment]
@rasheedamir please review if this looks fine: https://github.com/stakater/IngressMonitorController/blob/lean-readme/README.md

**Comment by stakater-user** [comment]
@ahmadiq Image is available for testing. ``docker pull stakater/ingressmonitorcontroller:PR-114-1``

File: docs/docs.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Documentation
+
+## Supported Uptime Checkers
```
Comment (rasheedamir) [inline docs/docs.md]: @ahmadiq lets keep this part in the README; just list of supported uptime checkers

Comment (ahmadiq) [reply docs/docs.md]: Done

**Comment by stakater-user** [comment]
@ahmadiq Image is available for testing. ``docker pull stakater/ingressmonitorcontroller:PR-114-2``

**Pull request merged**